src/test/java/org/ecommerce/services/jpa/impl/OrderJpaServiceImplTest.java

```diff
@@ -20,12 +20,17 @@
 import org.ecommerce.repositories.jpa.OrderJpaRepository;
```

Comment:
Unit tests of `OrderJpaServiceImplTest` are failing 👀

Reply:
Probably because of the Currency Converter API, looks like there's a limited number of requests per month, and it has been reached already. 
<img width="884" alt="image" src="https://github.com/user-attachments/assets/48691809-7459-45d6-a59e-1042b57e9ff8">
this is a new **API Key**  for the service : `14d55cc549d5fffbcb59ef6f2a37f92b`